src/extension/settings/settingsPropertyManager.ts

```diff
@@ -108,17 +108,28 @@ export class SettingsPropertyManager {
             profileObject = general.plainObjectToClass(ConanProfileConfiguration, selectedProfileObject);
         }
 
+        let pythonInterpreter = await python.getCurrentPythonInterpreter();
+        if (pythonInterpreter !== undefined) {
+            if (!profileObject?.conanPythonInterpreter) {
+                profileObject!.conanPythonInterpreter = pythonInterpreter;
```

Comment:
As far as i understood, is that this line of code will be used, in case the user doesn't set the python interpreter inside the configuration, so the extension will be smart enough to get the python interpreter from the python extension, really nice

But we must provide the visibility of this information to the user, from where the python interpreter is used during the usage of the extension, otherwise, user might be confused due to hidden information.

Reply:
- `README` is definitely the first step
- The extension output is also good to show it there (I see you already did that 😃)

I am thinking to put the information maybe in the tooltip information from the toolbar where the selected profile is shown. But if the two bullet points from above are there, i think we can move on.

I will take care of the rest, since it is more or less nice to have before the user gets confused, where the extension takes the python interpreter from 😄

Reply:
I like the tooltip idea! :-)
I've implemented.